### ISSUE TITLE
Error when sending Email - action is not authorized

### ISSUE THREAD

**Issue by tchang94** [CanMik/protocolsmanager#13]
Hello, 

I follow this page for the Email sending configuration of protocols manager. https://github.com/mateusznitka/protocolsmanager/wiki/Email-sending-configuration.

The server send Email correctly since a long time. (when i open / modify / close ticket)
I try the plugin manager since 2 month, but sending Email dosn't work and i dont know why. 

The file generation is OK. When i click "send", send Email page OK : 
"Select recipients from template or enter manually to send email :"
I can choose both choice :   Enter recipients manually or Select recipients from template

Same result, error message in french : "L'action que vous avez réalisée n'est pas autorisée." (the action is not authorized)

Could you help me, I didn't find where is my mistake ? please

Info : GLPI 10.0.9, Linux Debian
Way of sending emails: SMTP+TLS (xxxxxxxx@xxx.xxx@smtp.office365.com)

Regards,
Thomas

**Comment by tchang94** [comment]
Some precision : I created a template for email settings, and Enable email autosending is ON from the template setting.	
When i create the file, i receive the mail, but without the file generated. Is-it normal ? 

From user and protocols manager tab, Click on 'send email' buttom still doesn't work. >>> "L'action que vous avez réalisée n'est pas autorisée." (the action is not authorized)

**Comment by CanMik** [comment]
Hello have you tried after pulling the latest version after the PR of september 13th ?

**Comment by tchang94** [comment]
Hello,

Happy new year !
Much better with this version 1.5.3.7 ! Thank you,

E-mail worked and sended but whitout the PDF document. What’s wrong ?
Exemple in attachement.

Cdt,
Thomas TCHANG
01 74 74 06 25

De : Mikail ***@***.***>
Envoyé : mardi 31 décembre 2024 11:03
À : CanMik/protocolsmanager ***@***.***>
Cc : Thomas TCHANG ***@***.***>; Author ***@***.***>
Objet : Re: [CanMik/protocolsmanager] Error when sending Email - action is not authorized (Issue #13)


Hello have you tried after pulling the latest version after the PR of september 13th ?

**Comment by petertuharsky** [comment]
Hi, is there such a version? The Releases tab shows only 1.5.3.5

**Comment by tchang94** [comment]
![image](https://github.com/user-attachments/assets/2670edbf-6fe8-4cfc-9395-6c72861a9183)

**Comment by petertuharsky** [comment]
But
https://github.com/CanMik/protocolsmanager/releases

**Comment by tchang94** [comment]
When i download your link tar or zip, version is : '1.5.3.4'.

**Comment by CanMik** [comment]
When you pull the project, master branch is ahead of releases, I need to update releases

**Comment by CanMik** [comment]
Yes it's because I forgot to bump version, but master branch is updated and
releases are a bit outdated
